lxd/seccomp/seccomp.go

```diff
@@ -137,66 +138,67 @@ struct lxd_seccomp_data_arch {
 #define LXD_SECCOMP_NOTIFY_MOUNT 3
 #define LXD_SECCOMP_NOTIFY_BPF 4
 #define LXD_SECCOMP_NOTIFY_SCHED_SETSCHEDULER 5
+#define LXD_SECCOMP_NOTIFY_SYSINFO 6
 
 // ordered by likelihood of usage...
 static const struct lxd_seccomp_data_arch seccomp_notify_syscall_table[] = {
-	{ -1, LXD_SECCOMP_NOTIFY_MKNOD, LXD_SECCOMP_NOTIFY_MKNODAT, LXD_SECCOMP_NOTIFY_SETXATTR, LXD_SECCOMP_NOTIFY_MOUNT, LXD_SECCOMP_NOTIFY_BPF, LXD_SECCOMP_NOTIFY_SCHED_SETSCHEDULER },
+	{ -1, LXD_SECCOMP_NOTIFY_MKNOD, LXD_SECCOMP_NOTIFY_MKNODAT, LXD_SECCOMP_NOTIFY_SETXATTR, LXD_SECCOMP_NOTIFY_MOUNT, LXD_SECCOMP_NOTIFY_BPF, LXD_SECCOMP_NOTIFY_SCHED_SETSCHEDULER, LXD_SECCOMP_NOTIFY_SYSINFO},
```

Comment:
@stgraber I wasn't sure of some of the syscall numbers in this section. I looked at the kernel sources but wasn't entirely sure that I've matched the architectures of all of them correctly.